src/textual/widgets/_tabs.py

```diff
@@ -590,11 +591,17 @@ def watch_active(self, previously_active: str, active: str) -> None:
             underline.highlight_end = 0
             self.post_message(self.Cleared(self))
 
-    def _highlight_active(self, animate: bool = True) -> None:
+    def _highlight_active(
+        self,
+        animate: bool = True,
+        *,
+        animate_on_level: AnimationsEnum = AnimationsEnum.BASIC,
```

Comment:
Seems odd to have this in the signature. I don't see anywhere that sets it, and given it is private, the user should not need to set it.

Reply:
Fair point.